### PR TITLE
ui: Various CSS amends

### PR DESCRIPTION
**Pull request by johncowen**
This PR is a general CSS cleanup, mainly revolving around moving all the icon things out of %composite-row component

1. Split out %icon-definition from %composite-row
2. Add hover effect to node listing
3. Fix up proxy level list components
4. Rename our various pills to use a 100-900 scale
5. Reorganize other icon related things (consul-kind and
consul-external-source)

### CODE REVIEW

File: ui-v2/app/components/consul-external-source/index.hbs [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-      {{else}}
-        <span data-test-external-source={{externalSource}} class="consul-external-source {{externalSource}}">
-          {{#if label}}
-            {{label}}
+      <span data-test-external-source={{externalSource}} class="consul-external-source {{externalSource}}">
+        {{#if label}}
+          {{label}}
+        {{else}}
+          {{#if (eq externalSource 'aws')}}
+            Registered via {{uppercase externalSource}}
```
Comment (johncowen) [inline ui-v2/app/components/consul-external-source/index.hbs:9]: We no longer need those recursive components we had, so I removed all the extra complexity here

File: ui-v2/app/components/consul-kind/index.hbs [hunk 2]
```diff
@@ -86,10 +81,9 @@
           </dd>
         </dl>
       {{else}}
-          <span data-test-kind={{item.Kind}} class="consul-kind gateway">
-              <span>{{Name}}</span>
+          <span data-test-kind={{item.Kind}} class="consul-kind">
+            {{Name}}
```
Comment (johncowen) [inline ui-v2/app/components/consul-kind/index.hbs:85]: Again removed all the recursiveness from this component.

File: ui-v2/app/styles/components/filter-bar.scss [hunk 3]
```diff
@@ -27,19 +20,3 @@ html.template-acl.template-list .filter-bar [role='radiogroup'] {
...
-}
-%filter-bar .value-critical span::before {
-  @extend %with-cancel-square-fill-icon, %as-pseudo;
-}
-%filter-bar .value-allow span::before {
-  @extend %with-arrow-right-icon, %as-pseudo;
-}
-%filter-bar .value-deny span::before {
-  @extend %with-deny-icon, %as-pseudo;
-}
```
Comment (johncowen) [inline ui-v2/app/styles/components/filter-bar.scss:45]: We no longer need all these icons now we have the new filter bar dropdown filters.

File: ui-v2/app/styles/components/list-row/skin.scss [hunk 4]
```diff
@@ -21,3 +21,10 @@
 %list-row-detail {
   color: $gray-500;
 }
+%list-row-detail a {
+  color: inherit;
+}
+%list-row-detail a:hover {
+  color: $color-action;
+  text-decoration: underline;
+}
```
Comment (johncowen) [inline ui-v2/app/styles/components/list-row/skin.scss:30]: Any links in the detail portion of a row will now act like the node one (grey hover-to-blue)

File: ui-v2/app/templates/dc/services/show/upstreams.hbs [hunk 5]
```diff
@@ -26,8 +26,9 @@
   {{#let (sort-by (comparator 'service' sort) filtered) as |sorted|}}
       <ChangeableSet @dispatcher={{searchable 'service' sorted}} @terms={{search}}>
         <BlockSlot @name="set" as |searched|>
-          <ConsulServiceList
+          <ConsulUpstreamList
```
Comment (johncowen) [inline ui-v2/app/templates/dc/services/show/upstreams.hbs:29]: Yoikes! Some copypasta seems to have completely replaced the UpstreamList with the plain service one, this brings it back.

**Review by johncowen — COMMENTED** [review]
📓

**Review by kaxcode — APPROVED** [review]
Nice cleanup! ♻️ 🔥 🎨

**Pull request merged**